### ISSUE TITLE
Mega Menu Contrast Issue

### ISSUE THREAD

**Issue by JessWaggs** [UCSCLibrary/SiteImprove#38]
There seems to be an issue with how the mega menu loads where the SiteImprove tool perceives both the text of mega menu links and the background color to be white (which would obviously be a contrast issue!)

The link color for regular menu items is never white. The background color is white at default state and gold on hover.

<img width="926" alt="Screen Shot 2020-05-07 at 6 32 37 PM" src="https://user-images.githubusercontent.com/6412857/81360726-6bb67a00-9091-11ea-8d05-4a6bc312db1d.png">

**Comment by rschwab** [comment]
I've got a fix in for this on sitestage.

It basically just changes the color of the links in the submenu's to the color they'll be after classes are added.

There's a larger question about re-factoring this menu that we could have. Currently there's a lot of work being done in jquery that could be more smoothly handled with just css. This includes some of the work on making the site responsive. That's probably a conversation for a different place, but it came up as I worked on this.
